pkg/db/db.go

```
@@ -64,32 +71,52 @@ func createTable(ctx context.Context, pool *pgxpool.Pool, tableName string) erro
 		return err
 	}
 
 	_, err = pool.Exec(ctx, fmt.Sprintf(`
 		SELECT create_hypertable('%s', 'time', if_not_exists => true)
-	`, tableName))
+	`, ohclTable))
 	if err != nil {
 		return err
 	}
 
+	result, err := pool.Exec(ctx, fmt.Sprintf(`
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)